### ISSUE TITLE
Soft Inpainting conflict with sd-webui-ux

### ISSUE THREAD

**Issue by Hujikuio** [Haoming02/sd-webui-tabs-extension#26]
I know asking for compatibility with a custom theme is a bit much, so feel free to close this if this is not something you care to tackle. Just thought I'd bring it up if you did or it's a simple fix.

Everything works great in the txt2img tab, but in img2img, the soft inpainting accordian/tab disappears, and the scripts tab is empty, unlike the txt2img tab.

**Comment by Haoming02** [comment]
Can you show a screenshot? ~~so I don't need to install yet another UI~~

> Soft Inpainting accordian disappears

I thought this is in the parameters above the `Sampling method`? It shouldn't be affected by this Extension

**Comment by Hujikuio** [comment]
> Can you show a screenshot? ~so I don't need to install yet another UI~
> 
> > Soft Inpainting accordian disappears
> 
> I thought this is in the parameters above the `Sampling method`? It shouldn't be affected by this Extension

Yes, you're right, it's weird so I'm not sure where the conflict is occurring here. If I set it to "always show first tab", the soft inpainting will appear, but the other scripts still don't.

Here is the txt2img script tab working:
![image](https://github.com/user-attachments/assets/e97f0a5c-a733-4e21-a4ef-6be1683a6ccf)

Soft Inpaint missing:
![image](https://github.com/user-attachments/assets/6c5ea644-5fdf-4558-914d-52a1e0c56aa7)

Empty Scripts img2img tab:
![image](https://github.com/user-attachments/assets/30b8d870-451e-49a7-a04b-d5e2f0eb0299)

AR++ shows up but that's because it normally has an exception to be above the other scripts. (Which is also annoying, but completely unrelated and another ui-ux thing)

**Comment by Hujikuio** [comment]
Also I've tried messing around with different combinations of all the settings, so I don't think it is as simple as that unfortunately. Same with the tab extension initialization delay (I saw in a previous issue that fixed a different issue with Lobe theme.).

Like I said before though, if this ends up being some crazy complicated thing or is on ui-ux, it's no big deal. There's no perfect UI for me out there anyways lol (your extensions carry webui though)

**Comment by Hujikuio** [comment]
Also this is on Forge as you can probably tell by the screenshots, I should have mentioned that as it might be a Gradio 4 thing. ui-ux claims to support both gradio 3/4 webui a1111/forge though.

**Comment by Haoming02** [comment]
I'm installing `stable-diffusion-webui-ux-forge` to check it out

**Comment by Hujikuio** [comment]
Sounds good, I'm using the [extension variant](https://github.com/anapnoe/sd-webui-ux), which might be easier for you as well. Thought I would mention just in case, I can see how checking the other version for discrepancies would be helpful as well. Thank you.

**Comment by Haoming02** [comment]
Yeah, I also repro it

<p align="center"><img src="https://github.com/user-attachments/assets/167101db-5535-4b1e-8d0f-930a3befc415"></p>

**Comment by Haoming02** [comment]
Ah, I know why now. Go into this Extension folder, and edit the `tab_configs.csv` file to:

```csv
,txt,img
tabs,left,left
default,left,left
```

Basically, delete all other entires, and set both values of both `tabs` and `default` to **left**

**Comment by Haoming02** [comment]
**TL;DR:** This Extension by default puts the tabs under the gallery instead of the original position in the `img2img` tab, so that "scripts section" became empty. Putting them back there thus fixes the issue.

<hr>

Though, `Soft Inpainting` somehow also got a Tab button, and when using other Tab, the section is hidden in the parameters...

**Comment by Hujikuio** [comment]
Just tried and it works, thank you ❤️
